src/corecel/Types.hh

```diff
@@ -43,12 +51,12 @@ enum class MemSpace
     host,  //!< CPU memory
     device,  //!< GPU memory
     mapped,  //!< Unified virtual address space (both host and device)
+    size_,
```

Comment:
Because it adds 1 to the previous entry, which is `native` (which is aliased to a lower number!)  and so ends up being *not* the last one ... I originally did that and didn't even catch it immediately 😅